src/api/APIProxy.ts

```diff
@@ -140,48 +141,40 @@ class APIProxy {
     };
   };
 
-  signIn = (
+  signIn = async (
     variables: InferOptions<MUTATIONS["TokenAuth"]>["variables"],
     options?: Omit<InferOptions<MUTATIONS["TokenAuth"]>, "variables">
-  ) =>
-    new Promise<{ data: TokenAuth["tokenCreate"] }>(async (resolve, reject) => {
-      try {
-        this.client.resetStore();
-
-        const data = await this.fireQuery(
-          MUTATIONS.TokenAuth,
-          mutationData => mutationData!.tokenCreate
-        )(variables, {
-          ...options,
-          update: (proxy, updateData) => {
-            const handledData = handleDataErrors(
-              (tokenCreateMutationData: any) =>
-                tokenCreateMutationData.tokenCreate,
-              updateData.data,
-              updateData.errors
-            );
-            if (!handledData.errors && handledData.data) {
-              setAuthToken(handledData.data.token);
-              if (window.PasswordCredential && variables) {
-                navigator.credentials.store(
-                  new window.PasswordCredential({
-                    id: variables.email,
-                    password: variables.password,
-                  })
-                );
-              }
-            }
-            if (options && options.update) {
-              options.update(proxy, updateData);
-            }
-          },
-        });
-
-        resolve(data);
-      } catch (e) {
-        reject(e);
-      }
+  ): Promise<SignIn> => {
+    await this.client.resetStore();
+    let result: {
+      data: TokenAuth_tokenCreate | null;
+    } | null = null;
+
+    result = await this.fireQuery(
+      MUTATIONS.TokenAuth,
+      mutationData => mutationData!.tokenCreate
+    )(variables, {
+      ...options,
+      fetchPolicy: "no-cache",
```

Comment:
According to Apollo docs, this is the default/only behavior for mutations?

Reply:
It seems it's not the default, however we can make it default by passing `{ defaultOptions: { mutate: { fetchPolicy: "no-cache" } } }` to the ApolloClient constructor.

Reply:
`fetchPolicy: "no-cache"` stops apollo from updating cache, even for related objects 🤷‍♂️